components/admin/admin-accessibility.tsx

```
@@ -0,0 +1,231 @@
+          }
+        }
+        
+        if (e.key === 'Escape') {
+          // Allow escape key to break focus trap
+          const firstFocusable = e.currentTarget.querySelector(
+            'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])'
+          ) as HTMLElement;
+          firstFocusable?.blur();
+        }
+      }}
+    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)